tests/kafkatest/services/verifiable_producer.py

```diff
@@ -73,24 +73,27 @@ def __init__(self, context, num_nodes, kafka, topic, max_messages=-1, throughput
         self.acked_values = []
         self.not_acked_values = []
         self.produced_count = {}
-        self.prop_file = ""
+
+    def prop_file(self, idx):
```

Comment:
the reason is that producers in verifiable producers can be configured with different compression types, while all consumers in console consumer are the same (actually, from the comment in the console consumer, it is required that number of consumers in console consumer is 1).